docs/modules/servers/pages/distributed/context.adoc

```diff
@@ -0,0 +1,4 @@
+= James Distributed Server &mdash; Context
+:navtitle: Context
```

Comment:
I don't understand what "context" means.

Reply:
Ah, sorry. It is intended to provide the context in which this server was developed, i.e. the current state of emails and how this server fits into the landscape. It is intended for people who know nothing about James, and who are trying to understand what this is all about. It is very high-level.

There should be an example in the Basic Server. I was just trying to follow the same hierarchy.

Note that if you have any better ideas, we can certainly change the hierarchy / structure. I just think that we ought to maintain some consistency between the different Servers.